api/v1alpha2/zz_generated.conversion.go

```diff
@@ -1941,14 +1981,10 @@ func autoConvert_v1alpha3_VirtualMachineImageStatus_To_v1alpha2_VirtualMachineIm
 	out.ProviderContentVersion = in.ProviderContentVersion
 	out.ProviderItemID = in.ProviderItemID
 	out.Conditions = *(*[]v1.Condition)(unsafe.Pointer(&in.Conditions))
+	// WARNING: in.Type requires manual conversion: does not exist in peer-type
```

Comment:
JW - we have more leeway the image types, and in general for Status fields, but we're intentionally not bothering to restore this field on up version conversion? (We'd need to start do the save/restore obj in yaml trick we do for the VM).

Reply:
Good point! Added the restore logic in v1a1 and v1a2 conversion.

Reply:
Spoke offline and decided to drop this restore logic for now to avoid resetting the status field by `dst.Status = restored.Status`.